### ISSUE TITLE
Specification does not pass WCAG 2.0 Accessibility requirements

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#110]
The specification is not accessible. Need to fix. Shadi believes the issue is with Bikeshed and will check with Michael.

**Comment by WilcoFiers** [comment]
This took some manual work, but the problems were resolved.
